docs/ldml/tr35.md

```diff
@@ -2985,7 +3079,7 @@ The following table summarizes the syntax that can be used.
 | [[pat1]&[pat2]]      | The intersection of sets specified by pat1 and pat2 |
 | [[pat1]-[pat2]]      | The asymmetric difference of sets specified by pat1 and pat2 |
 | [a \{ab} \{ac}]      | The code point 'a' and the multi-code point strings "ab" and "ac" |
-| [x\\u\{61 2019 62}y] | Equivalent to [x\\u0061\\u2019\\u0062y] (= [xa’by]) |
+| [x\\u\{61 2019 62}y] | Equivalent to [x\\u0061\\u2019\\u0062y] (= [xa’by]) | <!-- TODO: Multi escape semantics? -->
```

Comment:
The way that multi-code-point escapes are defined, they are exactly equivalent to a series of single code point escapes. Having them magically convert to a string would break that.